script/build-newsletter-editions.js

```diff
@@ -173,6 +175,9 @@ async function saveNewsletterEditions(organizationId, letterheadData) {
           newsletter.title +
           "' was published at " +
           newsletter.publicationDate +
+          " '" +
```

Comment:
Same as above, looks [identical](https://github.com/news-catalyst/next-tinynewsdemo/blob/main/script/build-newsletter-editions.js#L178) to main.